### PR TITLE
auth frontend, regiter user

### PR DESCRIPTION
**Pull request by at-huydoan**
auth for frontend.

### CODE REVIEW

**Comment by at-huydoan** [comment]
@at-vienlm please review this PR

File: app/Http/Frontend/Controllers/Auth/AuthController.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    /**
+     * Where to redirect users after login / registration.
+     *
+     * @var string
+     */
+    protected $redirectTo = '/';
+
+    protected $redirectPath = '/';
+
+    protected $guard = 'web';
```
Comment (ghost) [inline app/Http/Frontend/Controllers/Auth/AuthController.php]: the name `web` is confused, I think `frontend` or null is better

File: app/Http/Frontend/Controllers/Auth/PasswordController.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    |
+    | This controller is responsible for handling password reset requests
+    | and uses a simple trait to include this behavior. You're free to
+    | explore this trait and override any methods you wish to tweak.
+    |
+    */
+
+    use ResetsPasswords;
+
+    protected $guard = 'web';
```
Comment (ghost) [inline app/Http/Frontend/Controllers/Auth/PasswordController.php]: please fix it,

**Comment by at-huydoan** [comment]
@at-vienlm bug was fixed

**Pull request merged**